tests/e2e/framework/kubernetes.go

```diff
@@ -505,18 +515,17 @@ func (k *KubeInfo) deployIstio() error {
 		istioYaml = mcNonAuthInstallFileNamespace
 	}
 	if *clusterWide {
-		if *authEnable {
-			istioYaml = authInstallFile
-		} else {
-			istioYaml = nonAuthInstallFile
-		}
+		istioYaml = getClusterWideInstallFile()
 	} else {
 		if *authEnable {
 			istioYaml = authInstallFileNamespace
 			if *multiClusterDir != "" {
 				istioYaml = mcAuthInstallFileNamespace
 			}
 		}
+		if *useGalleyConfigValidator {
```

Comment:
Maybe we should also try to merge all these logic into the helper function and call it getInstallFile()

Reply:
Well - we should remove all this logic and the config file ASAP. We have it just because the helm migration is not complete. 

The future is to use help template directly, and pass any customization as flags - either by doing the install of istio in a separate target before the e2e test, or by passing helm options to the install (so we test what users will really use, not the deprecated files)

Reply:
I think this is the best we can do in the current PR. Refactoring the test framework to invoke helm directly is a larger effort and outside the scope of this task.